tests/System.Buffers.Primitives.Tests/MemoryTests.cs

```diff
@@ -167,59 +167,6 @@ private static IEnumerable<object[]> MakePermutations(params object[][] argss)
         }
         #endregion
 
-        public static IEnumerable<object[]> ReservationPerformanceData =>
```

Comment:
Removing unused helpers in #1526 since this (now removed) test was the only one using them.